aspnetcore/signalr/background-services.md

```diff
@@ -8,47 +8,50 @@ ms.custom: mvc
+> For more information on strongly typed Hubs, see [Use hubs in SignalR for ASP.NET Core](xref:signalr/hubs#strongly-typed-hubs).
 
 [!code-csharp[Startup](background-service/sample/Server/ClockHub.cs?name=ClockHub)]
 
-The interface used by the strongly-typed `ClockHub` is the `IClock` interface.
+The interface used by the strongly typed `ClockHub` is the `IClock` interface.
 
 [!code-csharp[Startup](background-service/sample/HubServiceInterfaces/IClock.cs?name=IClock)]
 
-## Calling a SignalR Hub from a background service
+## Call a SignalR Hub from a background service
 
 During startup, the `Worker` class, a `BackgroundService`, is wired up using `AddHostedService`.
 
 ```csharp
 services.AddHostedService<Worker>();
 ```
 
-Since SignalR is also wired up during the `Startup` phase in which each Hub is attached to an individual endpoint in ASP.NET Core's HTTP request pipeline, each Hub is represented by an `IHubContext<T>` on the server. Using ASP.NET Core's dependency injection features, other classes instantiated by the hosting layer - like `BackgroundService` classes or MVC Controller classes or even Razor page models - can get references to server-side Hubs by accepting instances of  `IHubContext<ClockHub, IClock>` during construction.
+Since SignalR is also wired up during the `Startup` phase, in which each Hub is attached to an individual endpoint in ASP.NET Core's HTTP request pipeline, each Hub is represented by an `IHubContext<T>` on the server. Using ASP.NET Core's DI features, other classes instantiated by the hosting layer, like `BackgroundService` classes, MVC Controller classes, or Razor page models, can get references to server-side Hubs by accepting instances of `IHubContext<ClockHub, IClock>` during construction.
 
 [!code-csharp[Startup](background-service/sample/Server/Worker.cs?name=Worker)]
 
-As the `ExecuteAsync` method is called iteratively in the background service, the server's current date & time are sent to the connected clients using the `ClockHub`.
+As the `ExecuteAsync` method is called iteratively in the background service, the server's current date and time are sent to the connected clients using the `ClockHub`.
 
-## Reacting to SignalR events with Background Services
+## React to SignalR events with background services
 
-Using the [.NET Client for SignalR](/aspnet/core/signalr/dotnet-client), a `BackgroundService` or `IHostedService` implementor can also be used to connect to SignalR Hubs and respond to events, much like HTML or .NET client applications do using the JavaScript and .NET Clients for SignalR.
+Using the <xref:signalr/dotnet-client>, a `BackgroundService` or `IHostedService` implementor can also be used to connect to SignalR Hubs and respond to events, much like HTML or .NET client apps do using the JavaScript and .NET clients for SignalR.
```

Comment:
This sentence is hard to follow:
* Should "implementor" be "implementation"?
* Using the .NET client ... much like ... .NET client apps do ... using .NET clients [sounds circular]
* Could "can also be used to connect" be changed to "can connect"?
* "like HTML or .NET client apps do using the JavaScript and .NET clients for SignalR"
  * confusing combination of "or" and "and"
  * does this mean to say "like an HTML page can do by using the JavaScript client, and like a .NET desktop app can do by using the .NET client"?